pandas/core/indexes/datetimelike.py

```diff
@@ -61,18 +61,39 @@ def strftime(self, date_format):
         return np.asarray(self.format(date_format=date_format),
                           dtype=compat.text_type)
     strftime.__doc__ = """
+    Convert to specified date_format.
+
     Return an array of formatted strings specified by date_format, which
     supports the same string format as the python standard library. Details
     of the string format can be found in `python string format doc <{0}>`__
 
     Parameters
     ----------
     date_format : str
-        date format string (e.g. "%Y-%m-%d")
+        Date format string (e.g. "%Y-%m-%d").
 
     Returns
     -------
-    ndarray of formatted strings
+    numpy.ndarray
+        n-dimensional array of formatted strings
```

Comment:
I would just say "NumPy array" instead of "n-dimensional", as it is always 1D

Reply:
@jorisvandenbossche separate issue, this should actually return an ``Index`` here.